hangupsbot/plugins/cleverbot.py

```diff
@@ -248,69 +32,25 @@ def _handle_incoming_message(bot, event, command):
     if not bot.get_config_suboption(event.conv_id, 'cleverbot_percentage_replies'):
         return
 
+    if not cw:
+        return
+
     percentage = bot.get_config_suboption(event.conv_id, 'cleverbot_percentage_replies')
 
     if randrange(0, 101, 1) < float(percentage):
-        text = yield from cleverbot_ask(event.conv_id, event.text)
+        text = cw.say(event.text)
         if text:
             yield from bot.coro_send_message(event.conv_id, text)
 
 
 def chat(bot, event, *args):
-    """chat to Cleverbot"""
-    if args:
-        text = yield from cleverbot_ask(event.conv_id, ' '.join(args))
-        if not text:
-            text = _("<em>Cleverbot is silent</em>")
-
+    if not cw:
+        yield from bot.coro_send_message(event.conv_id, "API key not defined")
     else:
-        text = _("<em>you have to say something to Cleverbot</em>")
-
-    yield from bot.coro_send_message(event.conv_id, text)
+        text = cw.say(event.text)
```

Comment:
Use `*args` instead of `event.text`, so that you can pass in manual conversations (ie, from the `_handle_incoming_message`, and so that if the command is called manually, the text does not include the `/bot chat` part.